### DISCUSSION TITLE
优化窗口管理的右键菜单

### DISCUSSION THREAD

**Discussion by lizipeng0013** [linuxdeepin/developer-center#5721] [Features Request & Ideas | 特性请求 & 头脑风暴]
20.9的截图：
![VirtualBox_测试Deepin_20_09_2023_14_59_28](https://github.com/linuxdeepin/developer-center/assets/123794418/0b827e79-1545-41c1-ad1c-3cae9741ef07)
23 beta2的截图：
![VirtualBox_deepin23-beta2_20_09_2023_13_55_16](https://github.com/linuxdeepin/developer-center/assets/123794418/73a8f101-7b3a-4d49-8b16-ecdf7e363400)
23版本的菜单颜色纯白、间距变大、没有圆角、在白色窗口操作时边缘无阴影不易区分，和20版本相比真的是大倒退。希望多多优化，和系统整体更搭配

**Comment by BLumia** [comment]
是个回归问题吧， @babyfengfjx
